src/bag/models.py

```diff
@@ -82,12 +82,12 @@ class Meta:
 class Openbareruimte(GeoModel):
     id = models.CharField(max_length=16, primary_key=True, db_column="identificatie")
     document_mutatie = models.DateField(db_column="documentdatum")
-    document_nummer = models.CharField(max_length=20, db_column="documentnummer")
+    document_nummer = models.CharField(max_length=40, db_column="documentnummer") # FIXME: Remove after problem in sources for id: 0363300011951944 is fixed.
     begin_geldigheid = models.DateField(db_column="begingeldigheid")
     einde_geldigheid = models.DateField(null=True, db_column="eindgeldigheid")
     type = models.CharField(max_length=80, db_column="typeomschrijving")
     naam = models.CharField(max_length=80)
-    naam_nen = models.CharField(max_length=80, db_column="naamnen")
+    naam_nen = models.CharField(max_length=80, null=True, db_column="naamnen")
```

Comment:
Is dit een probleem met hetzelfde record? Of met maar één record? In zo'n geval kunnen we misschien beter dat record vangen in een try/except oid.

Reply:
Gaat om een wijziging in het model aan de kant van BenK